templates/bootstrap.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 set -ex
-sleep 30
```

Comment:
any reason why this `sleep` was removed? I remember we added it from Packer's warning (below) on their [doc page here](https://learn.hashicorp.com/tutorials/packer/getting-started-provision?in=packer/getting-started#configuring-provisioners).

![image](https://user-images.githubusercontent.com/7400326/116452247-826a5980-a82b-11eb-9cc8-82dbcb5ff032.png)